framework/tlvf/AutoGenerated/src/tlvf/test/tlvVarList.cpp

```diff
@@ -473,6 +473,10 @@ uint32_t& cInner::var1() {
     return (uint32_t&)(*m_var1);
 }
 
+std::string cInner::unknown_length_list_inner_str() {
+    return std::string(m_unknown_length_list_inner, m_unknown_length_list_inner_idx__);
```

Comment:
I believe that if the length is zero, m_unknown_length_list_inner will be nullptr. std::string constructor segfaults on that.

Would it be possible to add a tlvf_test to check this?

The fix is simply to check the length and return `std::string()` if it's empty.

Reply:
Fixed.

Reply:
Done